app/models/declaration/governmentagencygoodsitem/Commodity.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.declaration.governmentagencygoodsitem
+
+case class Commodity(description: Option[String] = None, // max 512 chars
```

Comment:
Do we need all these comments?
These should have been defined in the form or Spec.
If so, please remove the comments.

Reply:
will do